test/OpenTelemetry.Tests/Trace/TelemetrySpanTest.cs

```diff
@@ -67,5 +67,22 @@ public void CheckRecordExceptionEmpty()
             telemetrySpan.RecordException(null);
             Assert.Empty(activity.Events);
         }
+
+        [Fact]
+        public void ParentIds()
+        {
+            using var activitySource = new ActivitySource("testSource");
+            using var parentActivity = activitySource.CreateActivity("parentOperation", ActivityKind.Internal);
```

Comment:
without any listeners, this activity will not be created, so thats causing the unit test failures.

Reply:
Thanks! I'll take a look

Reply:
I believe I fixed this - didn't get any `dotnet test` failures back when running locally.